pkg/api/apis/operators/v1alpha1/clusterserviceversion_types.go

```diff
@@ -147,6 +162,7 @@ type ClusterServiceVersionSpec struct {
 	Maturity                  string                    `json:"maturity,omitempty"`
 	CustomResourceDefinitions CustomResourceDefinitions `json:"customresourcedefinitions,omitempty"`
 	APIServiceDefinitions     APIServiceDefinitions     `json:"apiservicedefinitions,omitempty"`
+	WebhookDefinitions        WebhookDefinitions        `json:"webhookdefinitions,omitempty"`
```

Comment:
I think this can just be a `[]WebhookDefinition` - not sure the "owned" distinction is useful here.

Reply:
I agree - I'll get around to that next.

Reply:
The `owned`  field has been removed.